### DISCUSSION TITLE
BinaryTransformation

### DISCUSSION THREAD

**Discussion by LindeNozomiLeo** [biomodhub/biomod2#282] [Q&A]
Hi,

How do I extract the raster or dataframe on which I can use the function bm_BinaryTransformation. I would like to get a binary map for my projections, however, these are stored as BIOMOD.projection.out.

I'm still very much getting used to how the data is stored on the harddrive, so any information on how to work with these i.e. load them into R (or view them) is very much welcome! (output.format=".img" seems to only save corrupt disk images)

**Comment by rpatin** [comment]
Hi @LindeNozomiLeo,
Thank you for reporting :pray: 
For support, we prefer to work with issues rather than discussion that we feel more appropriate for open discussion around the package.
I opened an issue related to your question: https://github.com/biomodhub/biomod2/issues/283
Best regards,
Rémi
